package.json

```diff
@@ -11,8 +11,8 @@
         "url": "git+https://github.com/padloc/padloc.git"
     },
     "engines": {
-        "node": "16.13.1",
-        "npm": "8.2.0"
+        "node": "16.x",
```

Comment:
I took the opportunity to tweak these because our Docker images also don't strictly force this minor/patch, and thus would generate warnings, unnecessarily.